docs/api/rest_api_extension_media_type.md

```diff
@@ -0,0 +1,92 @@
+# Adding a custom media type
```

Comment:
```suggestion
# Adding custom media type
```
this was not an error, but we try to keep articles out of headings per our style guide